app/assets/javascripts/angular/controllers/dashboard_ctrl.js

```diff
@@ -83,6 +83,10 @@ angular.module("Prometheus.controllers")
   };
 
   $document.keydown(function(ev) {
+    if (ev.keyCode === 90 && ev.metaKey) { // Z keycode + cmd key
```

Comment:
I would recommend not to use any command/ctrl key combinations in js. It seems to cause more trouble than good in my experience, as people will use a huge set of different shortcut combinations.

Feel free to ignore me, but I'd vote to keep promdash shortcuts simple and similar to gmail's shortcuts, so just `z` here.